TLM/TLM/TrafficManagerMod.cs

```diff
@@ -41,6 +41,21 @@ public class TrafficManagerMod : IUserMod {
 
         public string Description => "Manage your city's traffic";
 
+        internal static TrafficManagerMod Instance = null;
+
+        internal bool InGameHotReload { get; private set; } = false;
+
+        internal static AppMode currentMode => SimulationManager.instance.m_ManagersWrapper.loading.currentMode;
+
+        internal static bool InGame() {
```

Comment:
Would it not be better to use `SceneManager.GetActiveScene().name == "Game"`?

I've been testing that as part of #699 and it's proved extremely reliable. (Note: PR #699 is currently non-functional as the UI stuff is a big mess - I'm waiting to see if kvakvs new Form Builder stuff can be used rather than spending ages messing with raw CO UI)

Reply:
Not to mention that this will throw exception when used in menu (mentioned about that earlier)

Reply:
it was not possible from `OnCreated()`
Now that I moved my stuff into `Enable()` ... yes

Reply:
I have catched the exception so its fine :). I am going to change it to GetActiveScene() because its just cleaner that way

Reply:
@aubergine10 I suspect that `SceneManager.GetActiveScene().name == "Game"` is not as reliable as we thought... Problem with it is that you will get true inside `LoadingExtension.OnCreated()`... 
It seems that `LoadingManager.instance.m_loadingComplete` works way better for that case.